Configuration x5VideoPlayerType should use 'h5-page' instead of 'h5'

As the code show below:

https://github.com/joe223/chimee/blob/0f04e52355e9ba2316705a0982b134377f3b3729/src/config/video.ts#L246

```html
<video id="video" controls="" preload="" x5-video-player-type="h5" loop="loop" height="150" width="300">
        <source src="http://cdn.tencent.neigou.com/Public/Home/mobileAsset/images/tencent2018/video5.mp4" type="video/mp4">
</video>
```
That will render X5 video player unexpectedly.

We should set `x5-video-player-type` to `'h5-page'` Due to X5's documentation https://docs.qq.com/doc/DTUxGdWZic0RLR29B

## Expected Behavior【期望行为】

Page should render a video as browser-natived.

@toxic-johann Got an error

```shell
Cannot find module '../typings/base'
```

Is there any cmd or script to generate the missing `types` folder?

--- a/.eslintrc
+++ b/.eslintrc
@@ -21,6 +21,7 @@
     "sourceType": "module"
   },
   "rules": {
-    "valid-jsdoc": "off"
+    "valid-jsdoc": "off",
+    "prefer-promise-reject-errors": "off"
   }
 }

--- a/__tests__/chimee/video-attr.js
+++ b/__tests__/chimee/video-attr.js
@@ -272,9 +272,9 @@ describe('chimee video attributes', () => {
   test('x5VideoPlayerType', () => {
     expect(player.x5VideoPlayerType).toBe();
     expect(getAttr(videoElement, 'x5-video-player-type')).toBe(null);
-    player.x5VideoPlayerType = 'h5';
-    expect(player.x5VideoPlayerType).toBe('h5');
-    expect(getAttr(videoElement, 'x5-video-player-type')).toBe('h5');
+    player.x5VideoPlayerType = 'h5-page';
+    expect(player.x5VideoPlayerType).toBe('h5-page');
+    expect(getAttr(videoElement, 'x5-video-player-type')).toBe('h5-page');
     player.x5VideoPlayerType = false;
     expect(player.x5VideoPlayerType).toBe();
     expect(getAttr(videoElement, 'x5-video-player-type')).toBe(null);
@@ -284,7 +284,7 @@ describe('chimee video attributes', () => {
     expect(player.x5VideoPlayerType).toBe();
     expect(getAttr(videoElement, 'x5-video-player-type')).toBe(null);
     player.__dispatcher.videoConfigReady = false;
-    player.x5VideoPlayerType = 'h5';
+    player.x5VideoPlayerType = 'h5-page';
     expect(player.x5VideoPlayerType).toBe();
     expect(getAttr(videoElement, 'x5-video-player-type')).toBe(null);
   });

--- a/demo/playsinline/index.js
+++ b/demo/playsinline/index.js
@@ -7,7 +7,7 @@ const player = new Chimee({
   playsInline: true,
   preload: 'auto',
   x5VideoPlayerFullscreen: true,
-  x5VideoPlayerType: 'h5',
+  x5VideoPlayerType: 'h5-page',
   x5VideoOrientation: 'landscape',
   xWebkitAirplay: true,
   muted: true,

--- a/lib/index.browser.js
+++ b/lib/index.browser.js
@@ -3144,7 +3144,7 @@
 
 	    /**
 	     * equal to console.error, output `[${tag}] > {$msg}`
-	     * @param {string} tag optional, the header of log 
+	     * @param {string} tag optional, the header of log
 	     * @param {string} msg the message
 	     */
 
@@ -3168,7 +3168,7 @@
 	    }
 	    /**
 	     * equal to console.info, output `[${tag}] > {$msg}`
-	     * @param {string} tag optional, the header of log 
+	     * @param {string} tag optional, the header of log
 	     * @param {string} msg the message
 	     */
 
@@ -3198,7 +3198,7 @@
 	    }
 	    /**
 	     * equal to console.warn, output `[${tag}] > {$msg}`
-	     * @param {string} tag optional, the header of log 
+	     * @param {string} tag optional, the header of log
 	     * @param {string} msg the message
 	     */
 
@@ -3212,7 +3212,7 @@
 	    }
 	    /**
 	     * equal to console.debug, output `[${tag}] > {$msg}`
-	     * @param {string} tag optional, the header of log 
+	     * @param {string} tag optional, the header of log
 	     * @param {string} msg the message
 	     */
 
@@ -3226,7 +3226,7 @@
 	    }
 	    /**
 	     * equal to console.verbose, output `[${tag}] > {$msg}`
-	     * @param {string} tag optional, the header of log 
+	     * @param {string} tag optional, the header of log
 	     * @param {string} msg the message
 	     */
 
@@ -7784,12 +7784,12 @@
 	  x5VideoPlayerType: [accessor({
 	    set: function set(value) {
 	      if (!this.dispatcher.videoConfigReady) return value;
-	      var val = value === 'h5' ? 'h5' : undefined;
+	      var val = value === 'h5-page' ? 'h5-page' : undefined;
 	      this.dom.setAttr('video', 'x5-video-player-type', val);
 	      return value;
 	    },
 	    get: function get(value) {
-	      return this.dispatcher.videoConfigReady && value || (this.dom.getAttr('video', 'x5-video-player-type') ? 'h5' : undefined);
+	      return this.dispatcher.videoConfigReady && value || (this.dom.getAttr('video', 'x5-video-player-type') ? 'h5-page' : undefined);
 	    }
 	  })],
 	  xWebkitAirplay: [accessor({
@@ -11054,7 +11054,7 @@
 	      if (isObject(pluginConfig)) {
 	        pluginConfig.id = id;
 	      }
-	      var plugin = isFunction(pluginConfig) ? new pluginConfig({ id: id }, this, pluginOption) // eslint-disable-line 
+	      var plugin = isFunction(pluginConfig) ? new pluginConfig({ id: id }, this, pluginOption) // eslint-disable-line
 	      : new Plugin(pluginConfig, this, pluginOption);
 	      this.plugins[id] = plugin;
 	      _Object$defineProperty(this.vm, id, {

--- a/lib/index.esm.js
+++ b/lib/index.esm.js
@@ -3138,7 +3138,7 @@ var Log = function () {
 
     /**
      * equal to console.error, output `[${tag}] > {$msg}`
-     * @param {string} tag optional, the header of log 
+     * @param {string} tag optional, the header of log
      * @param {string} msg the message
      */
 
@@ -3162,7 +3162,7 @@ var Log = function () {
     }
     /**
      * equal to console.info, output `[${tag}] > {$msg}`
-     * @param {string} tag optional, the header of log 
+     * @param {string} tag optional, the header of log
      * @param {string} msg the message
      */
 
@@ -3192,7 +3192,7 @@ var Log = function () {
     }
     /**
      * equal to console.warn, output `[${tag}] > {$msg}`
-     * @param {string} tag optional, the header of log 
+     * @param {string} tag optional, the header of log
      * @param {string} msg the message
      */
 
@@ -3206,7 +3206,7 @@ var Log = function () {
     }
     /**
      * equal to console.debug, output `[${tag}] > {$msg}`
-     * @param {string} tag optional, the header of log 
+     * @param {string} tag optional, the header of log
      * @param {string} msg the message
      */
 
@@ -3220,7 +3220,7 @@ var Log = function () {
     }
     /**
      * equal to console.verbose, output `[${tag}] > {$msg}`
-     * @param {string} tag optional, the header of log 
+     * @param {string} tag optional, the header of log
      * @param {string} msg the message
      */
 
@@ -7778,12 +7778,12 @@ var accessorMap = {
   x5VideoPlayerType: [accessor({
     set: function set(value) {
       if (!this.dispatcher.videoConfigReady) return value;
-      var val = value === 'h5' ? 'h5' : undefined;
+      var val = value === 'h5-page' ? 'h5-page' : undefined;
       this.dom.setAttr('video', 'x5-video-player-type', val);
       return value;
     },
     get: function get(value) {
-      return this.dispatcher.videoConfigReady && value || (this.dom.getAttr('video', 'x5-video-player-type') ? 'h5' : undefined);
+      return this.dispatcher.videoConfigReady && value || (this.dom.getAttr('video', 'x5-video-player-type') ? 'h5-page' : undefined);
     }
   })],
   xWebkitAirplay: [accessor({
@@ -11048,7 +11048,7 @@ var Dispatcher = (_dec$7 = before(convertNameIntoId), _dec2$6 = before(convertNa
       if (isObject(pluginConfig)) {
         pluginConfig.id = id;
       }
-      var plugin = isFunction(pluginConfig) ? new pluginConfig({ id: id }, this, pluginOption) // eslint-disable-line 
+      var plugin = isFunction(pluginConfig) ? new pluginConfig({ id: id }, this, pluginOption) // eslint-disable-line
       : new Plugin(pluginConfig, this, pluginOption);
       this.plugins[id] = plugin;
       _Object$defineProperty(this.vm, id, {

--- a/lib/index.js
+++ b/lib/index.js
@@ -539,12 +539,12 @@ var accessorMap = {
   x5VideoPlayerType: [toxicDecorators.accessor({
     set: function set(value) {
       if (!this.dispatcher.videoConfigReady) return value;
-      var val = value === 'h5' ? 'h5' : undefined;
+      var val = value === 'h5-page' ? 'h5-page' : undefined;
       this.dom.setAttr('video', 'x5-video-player-type', val);
       return value;
     },
     get: function get(value) {
-      return this.dispatcher.videoConfigReady && value || (this.dom.getAttr('video', 'x5-video-player-type') ? 'h5' : undefined);
+      return this.dispatcher.videoConfigReady && value || (this.dom.getAttr('video', 'x5-video-player-type') ? 'h5-page' : undefined);
     }
   })],
   xWebkitAirplay: [toxicDecorators.accessor({
@@ -3313,7 +3313,7 @@ var Dispatcher = (_dec$6 = toxicDecorators.before(convertNameIntoId), _dec2$5 = 
       if (chimeeHelper.isObject(pluginConfig)) {
         pluginConfig.id = id;
       }
-      var plugin = chimeeHelper.isFunction(pluginConfig) ? new pluginConfig({ id: id }, this, pluginOption) // eslint-disable-line 
+      var plugin = chimeeHelper.isFunction(pluginConfig) ? new pluginConfig({ id: id }, this, pluginOption) // eslint-disable-line
       : new Plugin(pluginConfig, this, pluginOption);
       this.plugins[id] = plugin;
       _Object$defineProperty(this.vm, id, {

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
     "toxic-decorators": "^0.3.8"
   },
   "devDependencies": {
+    "babel-core": "^6.26.3",
     "babel-eslint": "^8.2.6",
     "babel-jest": "^23.4.2",
     "babel-plugin-dynamic-import-node": "^2.0.0",
@@ -118,6 +119,8 @@
     "transform": {
       ".*": "babel-jest"
     },
-    "setupFiles": ["jest-canvas-mock"]
+    "setupFiles": [
+      "jest-canvas-mock"
+    ]
   }
 }

--- a/src/config/video.js
+++ b/src/config/video.js
@@ -221,13 +221,13 @@ const accessorMap = {
     accessor({
       set(value) {
         if (!this.dispatcher.videoConfigReady) return value;
-        const val = value === 'h5' ? 'h5' : undefined;
+        const val = value === 'h5-page' ? 'h5-page' : undefined;
         this.dom.setAttr('video', 'x5-video-player-type', val);
         return value;
       },
       get(value) {
         return (this.dispatcher.videoConfigReady && value) ||
-          (this.dom.getAttr('video', 'x5-video-player-type') ? 'h5' : undefined);
+          (this.dom.getAttr('video', 'x5-video-player-type') ? 'h5-page' : undefined);
       },
     }),
   ],

--- a/src/dispatcher/index.js
+++ b/src/dispatcher/index.js
@@ -191,7 +191,7 @@ export default class Dispatcher {
       pluginConfig.id = id;
     }
     const plugin = isFunction(pluginConfig)
-      ? new pluginConfig({id}, this, pluginOption) // eslint-disable-line 
+      ? new pluginConfig({id}, this, pluginOption) // eslint-disable-line
       : new Plugin(pluginConfig, this, pluginOption);
     this.plugins[id] = plugin;
     Object.defineProperty(this.vm, id, {

